handlers/auth/login_task.go

```diff
@@ -143,6 +144,8 @@ func (LoginTask) Action(w http.ResponseWriter, r *http.Request) any {
 		log.Printf("login success uid=%d session=%s", row.Idusers, handlers.HashSessionID(session.ID))
 	}
 
+	stats.Inc("logins")
```

Comment:
@jules no nil check? This should not be a global it should be coming from cd. remove all globals